scrapers/FemJoy.yml

```diff
@@ -45,4 +58,46 @@ xPathScrapers:
       Performers: *performersAttr
       Image: //img[@class="lazy comment-photo"]/@data-original
       Details: *detailsSel
-# Last Updated October 24, 2021
+jsonScrapers:
+  performerSearch:
+    performer:
+      Name: results.#.name
+      URL:
+        selector: results.#.url
+        postProcess: &site
+          - replace:
+              - regex: ^
+                with: "https://femjoy.com"
+  performerScraper:
+    performer:
+      Name: item.name
+      Gender: #item.gender # api seems to have wrong male genders
+        fixed: female # use a fixed one instead
+      Birthdate: item.birth_date
+      Ethnicity: item.ethnicity
+      Height:
+        selector: item.height
+        postProcess:
+          - replace:
+              - regex: \\
+                with:
+          - feetToCm: true
+      Weight:
+        selector: item.weight
+        postProcess:
+          - replace:
+              - regex: \slbs
+                with:
+          - lbToKg: true
+      Measurements: item.cup_size
+      Image:
+        selector: item.thumb.image
+      Country: item.birth_place.country
```

Comment:
Yeah... the api endpoint is gone now though....
Back to the original  xPath scraper i guess ?

Reply:
It looks like they released a whole new site design. 

The old one is still accessible via `https://old.femjoy.com`, but API is gone. 

I made an xPATH scraper for scenes since that was broken too with the new design, someone else can expand on it if they want to. 

```
name: Femjoy
sceneByURL:
  - action: scrapeXPath
    url:
      - femjoy.com
    scraper: sceneScraper
xPathScrapers:
  sceneScraper:
    scene:
      Title: //div[@class='post_details']/h1[@class='post_title']/span/text()
      Studio: 
        Name: 
          fixed: Femjoy
      Date: 
        selector: //div[@class='post_details']/h2[@class='post_title']
        postProcess: 
          - replace:
              - regex: .+on\s(.+)
                with: $1
          - parseDate: Jan 2, 2006             
      Performers: 
        Name: //div[@class='post_details']/h1[@class='post_title']/a[contains(@href,"/models/")]
        URL: //div[@class='post_details']/h1[@class='post_title']/a[contains(@href,"/models/")]/@href        
      #Image: Uses placeholder cover now
      Details: //div[@class='post_details']/h2[@class='post_description']/p
# Last Updated October 13, 2022
```